packages/api/src/api/helix/channel/HelixChannelFollowedChannel.ts

```diff
@@ -0,0 +1,54 @@
+import { Enumerable } from '@d-fischer/shared-utils';
+import { checkRelationAssertion, DataObject, rawDataSymbol, rtfm } from '@twurple/common';
+import type { BaseApiClient } from '../../../client/BaseApiClient';
+import type { HelixChannelFollowedChannelData } from '../../../interfaces/helix/channel.external';
+import type { HelixUser } from '../user/HelixUser';
+
+/**
+ * Represents a broadcaster that a user follows.
+ */
+@rtfm<HelixChannelFollowedChannel>('api', 'HelixChannelFollowedChannel', 'broadcasterId')
+export class HelixChannelFollowedChannel extends DataObject<HelixChannelFollowedChannelData> {
```

Comment:
The duplication in the name feels weird, `HelixFollowedChannel` is probably fine. Not every result class name needs the Helix category as prefix.

Reply:
I thought so, too, but didn't want to break from convention without your review. Done.